seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -111,8 +115,6 @@ def test_list_workspaces(self, mock_time, mock_logger, mock_redis):
             _ = list_anvil_workspaces(user)
         self.assertEqual(str(ec.exception),
             'Error: called Terra API: GET /api/workspaces?fields=public,workspace.name,workspace.namespace got status: 401 with a reason: Unauthorized')
-        mock_logger.error.assert_called_with('GET https://terra.api/api/workspaces?fields=public,workspace.name,workspace.namespace 401 0 test_user')
-        self.assertEqual(len(mock_logger.method_calls), 1)
```

Comment:
I don't see any changes to `list_anvil_workspaces` so why is the test changing?

Reply:
I removed the log for [the Anvil response status code errors](https://github.com/broadinstitute/seqr/pull/1665/files#diff-8887af909d3e9ea336f2ed1b1ea6f94e0aad8485c254010ea4a9dba763b6d44aL103-L106) because the error message will be in the logs from the middleware while handling the exception.